openshift_tools/monitoring/zagg_sender.py

```diff
@@ -135,7 +131,7 @@ def add_heartbeat(self, heartbeat, host=None):
                                                  )
         self.unique_metrics.append(hb_metric)
 
-    def add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
+    def _add_zabbix_keys(self, zabbix_keys, host=None, synthetic=False):
```

Comment:
Is there a reason not to just rename this to `add_metric` directly instead of doing the forward method?